notebooks/slicing_tutorial_WiC.ipynb

```diff
@@ -0,0 +1,864 @@
+    "* `snorkel` is installed\n",
+    "* `PYTHONPATH` includes root of `snorkel-superglue`\n",
+    "* `SUPERGLUEDATA` is set where [download_superglue_data.py](https://github.com/HazyResearch/snorkel-superglue/blob/staging/download_superglue_data.py) was called."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys, os\n",
+    "import pandas as pd\n",
+    "from pathlib import Path\n",
+    "\n",
+    "from snorkel.mtl.data import MultitaskDataLoader\n",
+    "from snorkel.mtl.model import MultitaskModel\n",
+    "from snorkel.mtl.snorkel_config import default_config as config\n",
+    "from snorkel.mtl.trainer import Trainer\n",
+    "from snorkel.slicing.sf import slicing_function\n",
+    "from snorkel.slicing.apply import PandasSFApplier\n",
+    "from snorkel.slicing.utils import add_slice_labels, convert_to_slice_tasks\n",
+    "from snorkel.types import DataPoint\n",
+    "\n",
+    "if not \"cwd\" in globals():\n",
+    "    cwd = Path(os.getcwd())\n",
+    "sys.path.insert(0, str(cwd.parents[0]))\n",
+    "\n",
+    "import models\n",
+    "from dataloaders import get_dataset, get_dataloaders\n",
+    "from tokenizer import get_tokenizer\n",
+    "from utils import task_dataset_to_dataframe"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We'll be working with the [Words in Context (WiC) task](https://pilehvar.github.io/wic/). To start, let's look at a few examples."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Explore the WiC dataset\n",
+    "Let's take a look at a few examples! "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "os.environ['SUPERGLUEDATA'] = '/home/ubuntu/workspace/superglue_data/'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "task_name = \"WiC\"\n",
+    "data_dir = os.environ[\"SUPERGLUEDATA\"]\n",
+    "max_sequence_length = 256\n",
+    "bert_model = \"bert-large-cased\"\n",
+    "tokenizer_name = \"bert-large-cased\"\n",
+    "max_data_samples = None\n",
+    "batch_size = 4\n",
+    "\n",
+    "# Don't truncate the sentence when viewing examples\n",
+    "pd.set_option('display.max_colwidth', -1)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# load the word-piece tokenizer for the bert-large-cased vocabulary\n",
+    "tokenizer = get_tokenizer(tokenizer_name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the style of our MTL tutorial, we'll use a few helpers to load them into Pytorch datasets/our `MultitaskDataLoader`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
```

Comment:
Could you add a bit more explanation of what's happening, especially with externally-defined fns like `get_dataset`?  Can be inline comments